src/Integration/Binding/BindingController.cs

```diff
@@ -162,6 +165,20 @@ private void OnBindingFinished(ProjectInformation projectInformation, bool isFin
                 }
             }
         }
+
+        internal /*for testing purposes*/ void CheckForConflicts()
+        {
+            // TODO: implement something like: (need to think about perf, and UX part of that work
+            /*
+            var conflictsManager = this.ServiceProvider.GetService<IConflictsManager>();
+            var conflicts = conflictsManager.GetCurrentConflicts();
+
+            if (conflicts.Count > 0)
+            {
+                // Let the user know that he has conflicts
+                this.host.ActiveSection?.UserNotifications?.ShowNotificationWarning("Conflicts detected. Would you like to [fix them]()?", new Guid(), conflictsManager.GetResolveConflictsCommand(conflicts));
+            }*/
```

Comment:
Will replace this commented out code with a real code in next PR (no reason to delete it).